2020/5xxx/CVE-2020-5361.json

```diff
@@ -1,17 +1,70 @@
 {
```

Comment:
'https://www.dell.com/support/kbdoc/en-us/000180741/dsa-2020-119-dell-client-products-unauthorized-bios-password-reset-tool-vulnerability   ' does not match '^(ftp|http)s?://\\S+$'